build.gradle

```diff
@@ -43,59 +46,57 @@ buildscript {
+    implementation group: 'com.beust', name: 'jcommander', version: '1.82'
     //compile group: 'org.slf4j', name: 'slf4j-api', version: '1.7.25'
-    compile group: 'org.slf4j', name: 'jcl-over-slf4j', version: '1.7.25'
-    compile group: 'ch.qos.logback', name: 'logback-classic', version: '1.2.13'
-
-    // google grpc
-    compile group: 'io.grpc', name: 'grpc-netty', version: "${grpcVersion}"
-    compile group: 'io.grpc', name: 'grpc-protobuf', version: "${grpcVersion}"
-    compile group: 'io.grpc', name: 'grpc-stub', version: "${grpcVersion}"
-
-    compile group: 'com.googlecode.protobuf-java-format', name: 'protobuf-java-format', version: '1.4'
-    compile group: 'org.bouncycastle', name: 'bcprov-jdk18on', version: '1.78.1'
-    compile group: 'com.typesafe', name: 'config', version: '1.3.2'
-    compile "org.apache.commons:commons-lang3:3.14.0"
-    compile group: 'com.google.api.grpc', name: 'googleapis-common-protos', version: '0.0.3'
-    compile 'com.alibaba:fastjson:1.2.83'
-
-    compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.16.1'
-    compile group: 'org.projectlombok', name: 'lombok', version: '1.18.2'
-
-    compile group: 'org.jline', name: 'jline', version: '3.25.0'
-    compile 'io.github.tronprotocol:zksnark-java-sdk:1.0.0'
+    implementation group: 'org.slf4j', name: 'jcl-over-slf4j', version: '1.7.25'
+    implementation group: 'ch.qos.logback', name: 'logback-classic', version: '1.2.13'
+
+    // grpc
+    implementation group: 'io.grpc', name: 'grpc-netty', version: grpcVersion
+    implementation group: 'io.grpc', name: 'grpc-protobuf', version: grpcVersion
+    implementation group: 'io.grpc', name: 'grpc-stub', version: grpcVersion
+
+    // google protobuf
+    implementation group: 'com.google.protobuf', name: 'protobuf-java', version: protobufVersion
+    implementation group: 'com.google.protobuf', name: 'protobuf-java-util', version: protobufVersion
+
+    implementation group: 'com.google.code.gson', name: 'gson', version: '2.11.0'
+
+    implementation group: 'com.googlecode.protobuf-java-format', name: 'protobuf-java-format', version: '1.4'
+    implementation group: 'org.bouncycastle', name: 'bcprov-jdk18on', version: '1.78.1'
+    implementation group: 'com.typesafe', name: 'config', version: '1.3.2'
+    implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.14.0'
+    implementation group: 'com.alibaba', name: 'fastjson', version: '1.2.83'
+    implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.16.1'
+
+    compileOnly 'org.projectlombok:lombok:1.18.24'
+    annotationProcessor 'org.projectlombok:lombok:1.18.24'
+    testCompileOnly 'org.projectlombok:lombok:1.18.24'
+    testAnnotationProcessor 'org.projectlombok:lombok:1.18.24'
+
+    implementation group: 'org.jline', name: 'jline', version: '3.25.0'
+    implementation group: 'io.github.tronprotocol', name: 'zksnark-java-sdk', version: '1.0.0'
```

Comment:
It's best to keep these upgraded versions consistent with Java tron.

Reply:
The version of grpc、protobuf is same as that used in java-tron. Other dependency is not very important.